graphein/protein/tensor/io.py

```diff
@@ -360,7 +378,13 @@ def protein_df_to_tensor(
     positions[residue_indices, atom_indices] = torch.tensor(
         df[["x_coord", "y_coord", "z_coord"]].values
     ).float()
-    return positions
+
+    if return_coord_mask:

```

Comment:
I don't think this is necessary to return. Users can just use `coords == fill_value` for this.

Reply:
I think you cannot do it that way due to python not recognising these floats as the same value, so one has to reconstruct the coord mask via tolerance bounds to the fill value etc which introduces potential for errors